schema/app.config.yaml.schema.json

```diff
@@ -0,0 +1,157 @@
+{
```

Comment:
Super cool! 

Good idea @moritzraho. We could move it to aio-cli-lib-app-config if there are other libs we can think of that would need to use it, or we could keep it here given that, in my mind at least, the next probable use case for this schema would be doing validation on `aio app deploy`

Reply:
Good idea -- can we do this after this PR is merged? Since if not I will need to have another dependency for this PR, and testing it will be slightly inconvenient

Reply:
sure let's just keep track of it :)

Reply:
https://github.com/adobe/aio-cli-plugin-app/issues/667